test/change.test.js

```diff
@@ -104,6 +134,29 @@ describe('Change', function() {
       });
     });
 
+    describe('when a change doesnt exist - promise variant', function() {
+      beforeEach(function(done) {
+        var test = this;
+        Change.findOrCreateChange(this.modelName, this.modelId)
+        .then(function(result) {
+          test.result = result;
+          done();
+        })
+        .catch(function(err) {
+          return done(err);
+        });
```

Comment:
In the light of https://github.com/strongloop/loopback/pull/1999/files#r51441636, just simplify the `catch` part.

```js
.then(function(result) {
  // ...
})
.catch(done);
```